src/jsc/jsc_class.hpp

```diff
@@ -300,20 +300,43 @@ inline void ObjectWrap<ClassType>::get_property_names(JSContextRef ctx, JSObject
     }
 }
 
+static inline bool try_get_int(JSStringRef property, int64_t& value) {
+    value = 0;
+    auto str = JSStringGetCharactersPtr(property);
+    auto end = str + JSStringGetLength(property);
+    while (str != end && iswspace(*str)) {
+        ++str;
+    }
+    bool negative = false;
```

Comment:
You could use `int sign = 1`, assign `-1` below and avoid branch in line 322 (line 323: `value *= sign`).